torch/_subclasses/fake_tensor.py

```diff
@@ -534,10 +535,8 @@ def repeat_interleave_tensor(fake_mode, func, repeats, output_size=None):
         ):
             raise DynamicOutputShapeException(func)
 
-        from torch.fx.experimental.symbolic_shapes import constrain_range
-
         output_size = fake_mode.shape_env.create_unbacked_symint()
-        constrain_range(output_size, min=2, max=sys.maxsize - 1)
+        _constrain_range_for_size(output_size)
```

Comment:
@tugsbayasgalan I can't use the public torch.export.constrain_as_size here because it fails with "cannot get Int out of SymInt", because it dispatches to the C++ implementation of sym_constrain_range_for_size. The meta registration doesn't get hit because.... well, I don't know why you expected the meta reg to get hit, there's no tensor argument here!